cmd/gleaner/main.go

```diff
@@ -20,8 +22,25 @@ var viperVal, sourceVal, modeVal string
 var setupVal bool
 
 func init() {
-	log.SetFlags(log.Lshortfile)
-	// log.SetOutput(ioutil.Discard) // turn off all logging
+	// Output to stdout instead of the default stderr. Can be any io.Writer, see below for File example
+
+	// name the file with the date and time
+	const layout = "2006-01-02-15-04-05"
+	t := time.Now()
+	lf := fmt.Sprintf("gleaner-%s.log", t.Format(layout))
```

Comment:
you could probably combine this line and the next into 
`Logfile := fmt.Sprintf("gleaner-%s.log", t.Format(layout))`